javascript/react-verbs/src/common/hooks.ts

```diff
@@ -10,15 +10,21 @@ import type {
 import { identity, noop, num } from '@data-wrangling-components/primitives'
 import type { TableContainer } from '@essex/arquero'
 import { columnType, DataType } from '@essex/arquero'
+import type { IDropdownOption } from '@fluentui/react'
 import type ColumnTable from 'arquero/dist/types/table/column-table'
 import cloneDeep from 'lodash-es/cloneDeep.js'
 import set from 'lodash-es/set.js'
 import { useCallback, useEffect, useMemo, useState } from 'react'
 
-import type {
-	DropdownOptionChangeFunction,
-	StepChangeFunction,
-} from '../types.js'
+import type { StepChangeFunction } from '../types.js'
+
+// #region Dropdown Change Handler
+
+export type DropdownChangeHandler = (
+	event: React.FormEvent<HTMLDivElement>,
+	option?: IDropdownOption,
+	index?: number,
+) => void
```

Comment:
we have tons of inlined function typings for fluent handlers - maybe a central types/fluent.ts to contain these?

Reply:
Hmm, maybe, but I'm kind of a fan of colocating types with where they're primarily provided from. I'll have a think

Reply:
It doesn't make a ton of sense in context here, but I like this idea for an `essex-js-toolkit` package

Reply:
maybe so - I guess this app doesn't have a _ton_ of them directly, but in aggregate across our apps we do. It's actually probably something that should be pushed up to fluent though - they're just inconsistent about typings on function handlers.